### ISSUE TITLE
New version 0.92.0

### ISSUE THREAD

**Issue by arcangelweb** [kalanda/homeassistant-custom-components#1]
With the new version this magnificent component stops working, for now I managed to log the log, adding the following file.

**Test version 0.92.0b3**

https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

File: manifest.json

```
{
  "domain": "aemet",
  "name": "AEMET",
  "documentation": "https://github.com/kalanda/homeassistant-custom-components",
  "dependencies": [],
  "codeowners": ["@kalanda"],
  "requirements": []
}
```
> Log Details (WARNING)
> Tue Apr 23 2019 23:13:16 GMT+0200 (hora de verano de Europa central)
> Your configuration contains extra keys that the platform does not support.
> Please remove [name], [api_key], [station_id], [monitored_conditions].

**Comment by arcangelweb** [comment]
In version 0.91.4 it works correctly, but I also have installed the test version that is 0.92.0b3, which, given the changes that are being made, affects the component.

https://developers.home-assistant.io/docs/en/creating_integration_file_structure.html

Discuss my English is not my native language.

**Comment by arcangelweb** [comment]
I already manage that the component is executed with the final version 0.92.0

hass_folder/custom_components/aemet/

**File: manifest.json**
{
  "domain": "aemet",
  "name": "AEMET",
  "documentation": "https://github.com/kalanda/homeassistant-custom-components",
  "dependencies": [],
  "codeowners": ["@kalanda"],
  "requirements": []
}

**File:** `__init__.py`

This file creates it blank.

Do not make any modification of the component code and everything seems to work correctly.

**Comment by kalanda** [comment]
Thanks @arcangelweb 
This should be resolved with the changes integrated in this PR #2 thank to @outon contribution now available in this repository
